config/environment.js

```diff
@@ -76,5 +76,8 @@ module.exports = function(environment) {
   // Whether to load existing expData into the exp-frames
   ENV.loadData = false;
 
+  // Whether to validate survey forms
+  ENV.validate = false;
```

Comment:
As a future change, these key names are a bit ambiguous. We can, and probably should, make the feature flags easier to find (and easier to understand the meaning of) by grouping them together under a shared parent:

```
ENV.featureFlags = {
    loadData: false, 
    validate: false
}
```